--- a/tests/System/Service/SystemServiceTest.php
+++ b/tests/System/Service/SystemServiceTest.php
@@ -27,7 +27,7 @@
 
         $information = $service->information();
 
-        self::assertSame(PHP_VERSION, $information->versions->php->value);
+        self::assertSame(PHP_VERSION, $information->versions->php->version);
         self::assertSame($php->version(), $information->php->version);
         self::assertSame($php->extensions(), $information->php->extensions);
         self::assertSame($server->phpSapi(), $information->server->phpSapi);
